courses/free-courses-my.md

```diff
@@ -1,8 +1,20 @@
 ### Index
 
+* [Flutter](#flutter)
+* [MySQL](#mysql)
 * [Python](#python)
 
 
+### Flutter
+
+* [Flutter](https://www.youtube.com/playlist?list=PLUbA5XRGtepJZdgd6XMHF9-nPGQs57eys) - Htain Lin Shwe
+
+
+### MySQL
```

Comment:
maybe "databases" would be a better category?

Reply:
fixed the category please check @eshellman